app/src/main/java/nerd/tuxmobil/fahrplan/congress/models/DateInfos.java

```diff
@@ -20,27 +28,24 @@ public boolean sameDay(Moment today, int lectureListDay) {
     }
 
     /**
-     * Returns the index of today
-     *
-     * @param hourOfDayChange   Hour of day change (all lectures which start before count to the
-     *                          previous day)
-     * @param minuteOfDayChange Minute of day change
-     * @return dayIndex if found, -1 otherwise
+     * Returns the index of today.
+     * <p>
```

Comment:
remove p tag and extra asterisk on line 33.

Reply:
Thanks for spotting this. Resolved in bf2f42dd524f3959518f3c490b69c7ec082c3a4b.